posthog/api/action.py

```diff
@@ -331,13 +303,9 @@ def _serialize_entity(self, entity: Entity, filter: Filter, request: request.Req
             serialized.update(self._stickiness(filtered_events=filtered_events, filter=filter))
         return serialized
 
-    def _serialize_people(self, entity: Entity, people: QuerySet, request: request.Request) -> Dict:
+    def _serialize_people(self, people: QuerySet, request: request.Request) -> Dict:
         people_dict = [PersonSerializer(person, context={'request': request}).data for person in  people]
         return {
-            'action': {
```

Comment:
I literally just tried to do same in my PR. I think it would be great to do it, but in my case it broke clicking on a data point in the graph to see who did that action.

Reply:
Hmm. What was breaking? The people seem to load for me

Reply:
Apologies, I was thinking about the other serialize function, ignore.